openfl-gramine/MANUAL.md

```diff
@@ -2,16 +2,28 @@
 This manual will help you run OpenFL with Aggregator-based workflow inside SGX enclave with Gramine.
 
 ## Prerequisites
+
+This guide will take into account three different types of machines:
+- the building machine, where the docker image will be built;
+- the Aggregator machine, where the Aggregator will run;
+- the Collaborator machines, where the Collaborators will run.
+
+It is, by the way, not mandatory that these machines are physically different. A single machine can be used to carry out multiple functions; this choice is made only to make the following tutorial clearer.
+</br>
+In the following the requirements for the different machines will be presented.
+
 Building machine:
 - OpenFL
 - Docker should be installed, user included in Docker group
 
 Machines that will run an Aggregator and Collaborator containers should have the following:
+- OpenFL
```

Comment:
are you sure that's the case? as far as i understand, the docker image is used, so aggregator and collaborators using it don't really need openfl installed

Reply:
Yes, you're right that the training will be carried out with the Docker container, in which OpenFL will already be present. However, the user will still need the OpenFL functionalities available outside the Docker container to carry out the exchange and validation of the certificates, as reported in step 7 of the actual OpenFL-gramine workflow reported in Manual.md.